posthog/hogql_queries/experiments/test/test_trends_statistics_continuous.py

```diff
@@ -73,17 +84,18 @@ def run_test(stats_version, calculate_probabilities, are_results_significant, ca
                 self.assertAlmostEqual(intervals["test"][0], 80, delta=5)  # Lower bound
                 self.assertAlmostEqual(intervals["test"][1], 120, delta=5)  # Upper bound
             else:
-                # Original implementation behavior for small sample
-                self.assertAlmostEqual(probabilities[0], 0.5, delta=0.2)
-                self.assertAlmostEqual(probabilities[1], 0.5, delta=0.2)
-                self.assertEqual(significance, ExperimentSignificanceCode.LOW_WIN_PROBABILITY)
-                self.assertEqual(p_value, 1)
+                self.assertAlmostEqual(probabilities[0], 0.0, delta=0.005)
+                self.assertAlmostEqual(probabilities[1], 1.0, delta=0.005)
+                self.assertEqual(significance, ExperimentSignificanceCode.SIGNIFICANT)
+                self.assertAlmostEqual(p_value, 0.00049, delta=0.00001)
 
-                # Original implementation returns intervals as ratios/multipliers of the mean
-                self.assertAlmostEqual(intervals["control"][0], 1.0, delta=0.2)  # Lower bound is less than mean
-                self.assertAlmostEqual(intervals["control"][1], 1.2, delta=0.1)  # Upper bound is greater than mean
-                self.assertAlmostEqual(intervals["test"][0], 1.0, delta=0.2)
-                self.assertAlmostEqual(intervals["test"][1], 1.2, delta=0.1)
+                # Control: ~$100 mean with narrow interval due to old implementation
+                self.assertAlmostEqual(intervals["control"][0], 98, delta=3)
+                self.assertAlmostEqual(intervals["control"][1], 102, delta=3)
+
+                # Test: ~$105 mean with narrow interval due to old implementation
+                self.assertAlmostEqual(intervals["test"][0], 103, delta=3)
+                self.assertAlmostEqual(intervals["test"][1], 107, delta=3)
```

Comment:
Oh, I intentionally didn't change the v1 values previously.

Reply:
Yeah, I was a little confused here. But as the old implementation is also getting the "total" as input (what it gets from the query runner), not the mean, the test cases should be updated to reflect that. And hence the assertions had to be updated.

Does that make sense? I think the values in the assertions make more sense now as well.

Reply:
Yep, the explanation makes sense. Just flagging that I intentionally didn't change the behavior / return values for v1. I'm not strongly opposed to doing so, but the original intent was to keep v1 exactly how it was.